client/src/pages/MySitters/MySitters.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable prettier/prettier */
+import { useState, useEffect } from "react";
+import getRequests from '../../helpers/APICalls/getRequests';
+import { RequestData } from "../../interface/Request";
+import { Container, Grid, Box } from '@material-ui/core';
+import useStyles from "./useStyles";
+
+export default function MySitters(): JSX.Element {
+  const classes = useStyles();
+
+  const [requests, setRequests] = useState<RequestData[]>([]);
+
+  useEffect(() => {
+    getRequests()
+      .then(data => setRequests(data));
+  }, []);
+
+  console.log(requests)
```

Comment:
This can be removed, not a necessary log.

Reply:
Fixed